src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/ParameterNamesShouldMatchBaseDeclarationTests.Fixer.cs

```diff
@@ -49,7 +49,7 @@ public class TestClass : BaseClass
                               }",
                     },
                 },
-                NumberOfFixAllIterations = 2,
+                NumberOfFixAllIterations = 1,
```

Comment:
📝 This is the default value for this property. Now that the analyzer fixes everything in one pass, this property can be left at the default value and the test can potentially switch back to the simple form.